### PR TITLE
Add "Skip to Content" and "Skip to navigation" links.

### PR DESCRIPTION
**Pull request by fabianrbz**
## Description

Make the sidenav a11y friendly.

### CODE REVIEW

File: lib/nexmo_developer/app/views/layouts/partials/_footer.html.erb [hunk 1]
```diff
@@ -16,11 +16,6 @@
       </div>
     </div>
 
-      <% if info[:status] %>
```
Comment (mheap) [inline lib/nexmo_developer/app/views/layouts/partials/_footer.html.erb:19]: Was this intentionally removed?

Comment (fabianrbz) [reply lib/nexmo_developer/app/views/layouts/partials/_footer.html.erb]: no, bad commit 🤦

**Review by mheap — APPROVED** [review]

**Comment by bencgreenberg** [comment]
Is this waiting for something before merging?

**Comment by fabianrbz** [comment]
@bencgreenberg @mheap this should be ready

**Pull request merged**